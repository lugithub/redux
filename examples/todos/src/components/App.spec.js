import React from 'react'
import { Provider } from 'react-redux'
import { createStore } from 'redux'
import TestRenderer from 'react-test-renderer'
import reducer from '../reducers'
import App from './App'

const store = createStore(reducer)

describe('App component', () => {
  it('should render tree', () => {
    const tree = TestRenderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
})
