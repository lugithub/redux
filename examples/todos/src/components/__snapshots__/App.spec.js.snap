// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render tree 1`] = `
<div>
  <div>
    <form
      onSubmit={[Function]}
    >
      <input />
      <button
        type="submit"
      >
        Add Todo
      </button>
    </form>
  </div>
  <ul />
  <p>
    Show:
     
    <span>
      All
    </span>
    , 
    <a
      href="#"
      onClick={[Function]}
    >
      Active
    </a>
    , 
    <a
      href="#"
      onClick={[Function]}
    >
      Completed
    </a>
  </p>
</div>
`;
